src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -226,6 +230,12 @@ protected int doWork() {
         if (WRITE_ORIGINAL_POSITION) {
             for (final VCFInfoHeaderLine line : ATTRS) outHeader.addMetaDataLine(line);
         }
+
+        outHeader.addMetaDataLine(new VCFInfoHeaderLine(LiftoverUtils.SWAPPED_ALLELES, 0, VCFHeaderLineType.Flag,
+                "The REF and the ALT alleles have been swapped due to changes in the reference. " +
+                "NOTA BENE: No INFO field annotations reflect this swap, and in the genotypes, " +
```

Comment:
Well, TAGS_TO_REVERSE do reflect this swap...

Reply:
right....fixing.